.github/workflows/main.yml

```diff
@@ -249,18 +249,6 @@ jobs:
       - name: Commit Validation tests
         run: .github/retry.sh ant $OPTS -Dcluster.config=$CLUSTER_CONFIG commit-validation
 
-      - name: Set up JDK 11 for core.network
-        uses: actions/setup-java@v4
-        if: matrix.java == 17 && success()
-        with:
-          java-version: 11
-          distribution: ${{ env.default_java_distribution }}
-
-      # TODO fails on 17+
-      - name: platform/core.network
-        if: matrix.java == 17 && success()
-        run: ant $OPTS -f platform/core.network test
-
```

Comment:
i don't want to distract from the main goal of this PR by pointing out trivialities, but the network test was under CV so that it can run on win/mac too. Is there a reason why it got moved to the linux only platform job?

Reply:
thanks